README.md

```diff
@@ -8,9 +8,26 @@ Some new use cases for Web technology have motivated thinking about a multi-reso
 
 Local sharing is quite popular, especially in Emerging Markets countries, due to cost and limitations on cellular data and relatively spotty WiFi availability. It is typically done over local Bluetooth/WiFi, by either built-in OS features like [Android Beam](https://en.wikipedia.org/wiki/Android_Beam) or with popular 3-rd party apps, such as [ShareIt](https://play.google.com/store/apps/details?id=com.lenovo.anyshare.gps) or [Xender](https://play.google.com/store/apps/details?id=cn.xender)). Typically, the locally stored media files and apps (APK files for Android for example) are shared this way. Extending sharing to bundles of content and web apps (Progressive Web Apps in particular) opens up new possibilities, especially if combined with a form of signing the content. Cryptographic signing could make it possible to afford the shared content the treatment normally reserved for the origin that content is claiming to be in - by verifying that the content indeed was produced by the holder of the corresponding certificate.
 
+Local sharing tends to have persistent constraints on the amount of data
+transferred: each byte costs money. This means the client may be stuck with an
+outdated version of a package for a significant amount of time, including that
+package's vulnerabilities. It may be feasible to periodically check for:
```

Comment:
Seems to me that rewording this without the numbered list would be clearer.  "It may be feasible to periodically check for OCSP notification that a package's certificate has been revoked.  Other channels for notifying about vulnerable packages have not yet been discussed/designed."